src/useRequestExposureNotifications.tsx

```diff
@@ -13,46 +13,34 @@ export const useRequestExposureNotifications = (): (() => void) => {
 
   const requestExposureNotifications = async () => {
     const response = await NativeModule.requestAuthorization()
-    if (response.kind === "success") {
-      handleENRequestSuccess(response.status)
-    } else {
-      handleENRequestFailure(response.error)
-    }
-  }
 
-  const handleENRequestSuccess = (status: ENPermissionStatus) => {
-    switch (status) {
-      case "Active":
-        break
-      case "BluetoothOff":
-        showEnableBluetoothAlert()
-        break
-      default:
-        showBaseExposureNotificationsAlert()
-    }
-  }
-
-  const handleENRequestFailure = (error: RequestAuthorizationError) => {
-    switch (error) {
-      case "Restricted":
-        showSetToActiveRegionAlert()
-        break
-      case "NotAuthorized":
-        showBaseExposureNotificationsAlert()
-        break
-      case "Unknown":
-        showBaseExposureNotificationsAlert()
-        break
-      default:
-        showBaseExposureNotificationsAlert()
-    }
-  }
-
-  const showBaseExposureNotificationsAlert = () => {
-    if (Platform.OS === "ios") {
-      showShareExposureInformationAlert()
+    if (response.kind === "success") {
+      switch (response.status) {
```

Comment:
@mattThousand I'm confused as to why `BluetoothOff` is under the "success" column here, but I figured `LocationOff` should go here as well. Should this be refactored so that anything but `Active` is considered a "failure"?

These types seem weird to me considering the overlap:

```
type RequestAuthorizationError =
  | "IOSUnknown"
  | "BadParameter"
  | "NotEntitled"
  | "NotAuthorized"
  | "Unsupported"
  | "Invalidated"
  | "BluetoothOff"
  | "InsufficientStorage"
  | "NotEnabled"
  | "APIMisuse"
  | "Internal"
  | "InsufficientMemory"
  | "RateLimited"
  | "Restricted"
  | "BadFormat"
  | "DataInaccessible"
  | "TravelStatusNotAvailable"
  | "Unknown"

type ENPermissionStatus =
  | "Unknown"
  | "Active"
  | "Disabled"
  | "BluetoothOff"
  | "LocationOff"
  | "Restricted"
  | "Paused"
  | "Unauthorized"
```

Reply:
@devinjameson I agree the overlap is confusing; there are 2 separate concepts at play; one is the "status" of the `Exposure Notifications` framework, for which `BluetoothOff` is a valid state (neither a "success" nor a failure persay) and the other concept is the error that we might get back when attempting to activate ENs. 

For example, the `EN` framework could be in the `BluetoothOff` state, in which case if we try to activate it, we will get a `BluetoothOff` error response.

Reply:
perhaps it makes sense to update the api for `requestAuthorization` to fulfill with only a 'success' or reject with the error?

Reply:
good points, I agree that the overlap is confusing. I think the intention in the current construction is to model the OS api; our JS method ultimately calls through to `setExposureNotificationEnabled` (https://developer.apple.com/documentation/exposurenotification/enmanager/3583729-setexposurenotificationenabled), which returns a `success` even in the case when `exposureNotificationStatus` is `bluetoothOff`

Reply:
https://github.com/Path-Check/gaen-mobile/pull/732